app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/base/RtcCall.kt

```diff
@@ -17,29 +17,20 @@ import org.json.JSONObject
 import org.webrtc.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/base/RtcCall.kt#L17"></span>org.webrtc.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>